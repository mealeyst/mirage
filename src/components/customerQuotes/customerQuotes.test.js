import React from 'react'
import { css } from 'styled-components'
import 'jest-styled-components'

import { CustomerQuotes } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) CustomerQuotes', () => {
  const createCustomerQuotes = (props) => {
    return shallowWithTheme(<CustomerQuotes {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createCustomerQuotes())
    .toMatchSnapshot()
  })
})
