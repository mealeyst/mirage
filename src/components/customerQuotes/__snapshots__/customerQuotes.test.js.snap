// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) CustomerQuotes matching the snapshot 1`] = `
.c0 {
  margin-top: 4rem;
  margin-bottom: 2rem;
}

.c0 .c1,
.c0 .c2,
.c0 .sc-uJMKN {
  text-align: center;
}

.c0 .c2,
.c0 .sc-uJMKN {
  color: #0073D1;
  margin-top: 0;
  margin-bottom: 0;
  font-style: italic;
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c0 blockquote {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
}

@media (min-width:76.8em) {
  .c0 {
    margin-top: 6rem;
  }
}

@media (min-width:76.8em) {
  .c0 .c2,
  .c0 .sc-uJMKN {
    -webkit-order: 2;
    -ms-flex-order: 2;
    order: 2;
  }
}

@media (max-width:958px) {
  .c0 .c1 {
    font-size: 3.4rem;
  }
}

<BaseCustomerQuotes
  className="c0"
  constrained={true}
  header="WHAT AWESOME PARENTS (LIKE YOU!) ARE SAYING"
  padding={true}
  quotes={
    Array [
      Object {
        "quote": "“These are real clothes to LIVE and PLAY in! My son is full of personality and wears Rockets of Awesome to reflect that.”",
        "signature": "Andrea, Dallas TX",
      },
      Object {
        "quote": "“My spunky daughter loves to be an original and love that we've found a brand that can help her define her unique sense of self!”",
        "signature": "Maddie, Cincinnati OH",
      },
      Object {
        "quote": "“My son loves your stuff more than any of his other clothes, and digs through his drawers to find them! I’m not sure if it’s the super-softness or the graphics, but thank you!”",
        "signature": "Courtney, Philadelphia PA",
      },
      Object {
        "quote": "“It’s great to see my kids feel so confident that what they’re wearing expresses who they are as individuals.”",
        "signature": "Robin, Brooklyn NY",
      },
    ]
  }
/>
`;
